frontend/src/components/Notificacao/Evolucao/Form.vue

```diff
@@ -0,0 +1,120 @@
+              v-mask="'##/##/#### ##:##'"
+              validate-on-blur
+              :rules="rules.dataHoraNotificacao"
+              required
+            />
+          </v-col>
+        </v-row>
+        <v-row>
+          <v-col>
+            <v-autocomplete
+              :value="localIsolamento"
+              :rules="rules.localIsolamento"
+              label="Local do isolamento *"
+              :items="locais.items"
+              item-text="value"
+              item-value="key"
+              :loading="locais.loading"
+              no-data-text="Local de isolamento não encontrado"
+            />
+          </v-col>
+        </v-row>
+        <v-row>
+          <v-col>
+            <v-autocomplete
+              :value="situacaoPaciente"
+              :rules="rules.situacaoPaciente"
+              label="Local do isolamento *"
+              :items="situacoes.items"
+              item-text="value"
+              item-value="key"
+              :loading="situacoes.loading"
+              no-data-text="Situação do paciente não encontrada"
+            />
+          </v-col>
+        </v-row>
+        <v-card-actions>
+          <v-row align="center" justify="end">
+            <v-col cols="auto">
+              <v-btn color="primary" rounded>ATUALIZAR NOTIFICÃO</v-btn>
+            </v-col>
+          </v-row>
+        </v-card-actions>
+      </v-form>
+    </v-container>
+  </v-card>
+</template>
+<script>
+import { required, dateHourMinuteFormat } from '@/validations/CommonValidations';
+import { mask } from 'vue-the-mask';
+
+const LOCAIS = [
+  { key: 'ALTA_ISOLAMENTO_DOMICILIAR', value: 'Alta com isolamento domiciliar' },
+  { key: 'INTERNAMENTO_LEITO_COMUM', value: 'Hospitalizado – Leito comum' },
+  { key: 'INTERNAMENTO_LEITO_UTI', value: 'Hospitalizado - Leito UTI' },
+];
+
+const SITUACOES = [
+  { key: 'SUSPEITO', value: 'Suspeito' },
+  { key: 'CONFIRMADO', value: 'Confirmado' },
+  { key: 'DESCARTADO', value: 'Descartado' },
+  { key: 'CURA', value: 'Curado' },
+  { key: 'ENCERRADO', value: 'Encerrado' },
+  { key: 'OBITO', value: 'Óbito' },
+];
+
+export default {
+  directives: { mask },
+  data: () => ({
+    valid: true,
+    dataHoraNotificacao: '',
+    localIsolamento: '',
+    situacaoPaciente: '',
+    locais: {
+      items: [],
+      loading: true,
+    },
+    situacoes: {
+      items: [],
+      loading: true,
+    },
+    rules: {
+      dataHoraNotificacao: [required, dateHourMinuteFormat],
+      localIsolamento: [required],
+      situacaoPaciente: [required],
+    },
+  }),
+  methods: {
+    loadLocais() {
+      this.locais.loading = true;
+      setTimeout(() => {
+        this.locais.items = LOCAIS;
+        this.locais.loading = false;
+      }, 2000);
+    },
+    loadSituacoes() {
+      this.situacoes.loading = true;
+      setTimeout(() => {
+        this.situacoes.items = SITUACOES;
+        this.situacoes.loading = false;
+      }, 2000);
+    },
+  },
+  mounted() {
```

Comment:
Aqui também, melhor usar no created
```suggestion
  created() {
```